pkgs/tools/system/osquery/default.nix

```diff
@@ -60,7 +32,9 @@ stdenvNoCC.mkDerivation rec {
 
   pname = "osquery";
 
-  inherit src version;
+  version = info.osquery.rev;
+
+  src = fetchFromGitHub info.osquery;
```

Comment:
I think it would be easier to understand from where things are fetched if only the hash came out of the json and the other parameters would remain in nix.